easybuild/easyblocks/generic/rpackage.py

```diff
@@ -155,7 +155,8 @@ def build_step(self):
     def install_R_package(self, cmd, inp=None):
         """Install R package as specified, and check for errors."""
 
-        cmdttdouterr, _ = run_cmd(cmd, log_all=True, simple=False, inp=inp, regexp=False)
+        cmd_full = "%s %s" % (self.cfg['prebuildopts'], cmd)
```

Comment:
@jdonners Shouldn't this be `preinstallopts` (since the command being executed here is `R CMD INSTALL`?

Also, make it aware of `installopts` too while you're at it?

Reply:
@jdonners ping?

Reply:
@jdonners re-ping?

Reply:
@jdonners ping? :)

Reply:
sorry, I should better check the issues where I'm mentioned..

is there even a separate build step for an R package? I could add preinstallopts like:
```python
cmd_full = "%s %s %s" % (self.cfg['prebuildopts'], self.cfg['preinstallopts'], cmd)
```
would that work?